### DISCUSSION TITLE
26.26.26.1关于–这是什么

### DISCUSSION THREAD

**Discussion by specked2527** [localsend/localsend#1040] [L-简体中文] [answered]
26.26.26.1关于–这是什么 ？
通过链接发送显示的
希望添加到解释说明

**Comment by k5746** [comment, marked as answer]
请将 LocalSend 添加到代理软件的白名单。

参考：
- [配置文件vpnservice中26.26.26.1和26.26.26.2这样的地址是什么意思呢](https://github.com/xiaokangwang/AndroidLibV2ray/issues/1)
- [26.26.26.1 这个 IP，有人知道这个 IP 地址是什么的吗 - V2EX](https://www.v2ex.com/t/378224)
